components/prism-turtle.js

```diff
@@ -0,0 +1,52 @@
+Prism.languages.turtle = {
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'multiline-string': {
+		pattern: /"""(?:[^"]"?"?)*"""|'''(?:[^']'?'?)*'''/,
+		greedy: true,
+		alias: 'string',
+		inside: {
+			'comment': /#.*/
+		}
+	},
+	'string': {
+		pattern: /"(?:[^\\"\r\n]|\\.)*"|'(?:[^\\'\r\n]|\\.)*'/,
+		greedy: true
+	},
+	'url': {
+		pattern: /<[^\s<>]*>/,
+		greedy: true,
+		inside: {
+			'punctuation': /[<>]/
+		}
+	},
+	'function': {
+		pattern: /(?:(?![-.\d\xB7])[-.\w\xB7\xC0-\uFFFD]+)?:(?:(?![-.])(?:[-.:\w\xC0-\uFFFD]|%[\da-f]{2}|\\.)+)?/i,
+		inside: {
+			'prefix': {
```

Comment:
This won't work if the local name contains colons, e.g. `foo:bar:baz`.

Use this instead:

```js
{
	'local-name': {
		pattern: /([^:]*:)[\s\S]+/,
		lookbehind: true
	},
	'prefix': {
		pattern: /[\s\S]+/,
		inside: {
			'punctuation': /:/
		}
	}
}
```

Reply:
Thanks, fixed